bitnami/mongodb/Chart.yaml

```diff
@@ -39,4 +39,4 @@ maintainers:
 name: mongodb
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/mongodb
-version: 15.1.7
+version: 15.1.8
```

Comment:
Bump a minor version when adding new values --> 15.2.0

Reply:
Hi @dgomezleon,
I have updated the version accordingly.